lib/rspec/github/formatter.rb

```diff
@@ -8,13 +8,26 @@ module Github
     class Formatter < RSpec::Core::Formatters::BaseFormatter
       RSpec::Core::Formatters.register self, :example_failed, :example_pending
 
+      def self.relative_path(path)
+        if (workspace = ENV['GITHUB_WORKSPACE'])
+          workspace = "#{File.realpath(workspace)}#{File::SEPARATOR}"
+          absolute_path = File.realpath(path)
+
+          return absolute_path.delete_prefix(workspace) if absolute_path.start_with?(workspace)
+        end
+
+        path
+      end
```

Comment:
I actually find this quite hard to read / understand (especially since there are no tests). So that's my first remark: Please add some test cases.

And what if you write something like this (completely untested):

```suggestion
      def self.relative_path(relative_path)
        absolute_path = File.realpath(relative_path)
        workspace = File.realpath(ENV.fetch('GITHUB_WORKSPACE', '.'))
        absolute_path.delete_prefix(workspace)
      end
```

But that's where tests will hopefully ensure that this stuff I wrote actually does any good